app/helpers/admin/edition_actions_helper.rb

```diff
@@ -1,93 +1,4 @@
 module Admin::EditionActionsHelper
-  def edit_edition_button(edition)
-    link_to "Edit draft", edit_admin_edition_path(edition), title: "Edit #{edition.title}", class: "btn btn-default btn-lg add-left-margin"
-  end
-
-  def redraft_edition_button(edition)
-    button_to "Create new edition to edit", revise_admin_edition_path(edition), title: "Create new edition to edit", class: "btn btn-default btn-lg"
-  end
-
-  def content_data_button(edition)
-    url = content_data_page_data_url(edition)
-
-    link_to "View data about page",
-            url,
-            class: "btn btn-default btn-lg pull-right"
-  end
-
-  def approve_retrospectively_edition_button(edition)
-    confirmation_prompt = "Are you sure you want to retrospectively approve this document?"
-    tag.div(class: "approve_retrospectively_button") do
-      capture do
-        form_for [:admin, edition], url: approve_retrospectively_admin_edition_path(edition, lock_version: edition.lock_version), method: :post do |form|
-          concat(form.submit("Looks good", data: { confirm: confirmation_prompt }, class: "btn btn-success"))
-        end
-      end
-    end
-  end
-
-  def submit_edition_button(edition)
-    button_to "Submit for 2nd eyes", submit_admin_edition_path(edition, lock_version: edition.lock_version), class: "btn btn-success second-eyes"
-  end
-
-  def reject_edition_button(edition)
-    button_to "Reject", reject_admin_edition_path(edition, lock_version: edition.lock_version), class: "btn btn-warning"
-  end
-
-  def publish_edition_form(edition, options = {})
-    button_title = "Publish #{edition.title}"
-
-    if options[:force]
-      link_to(
-        "Force publish",
-        confirm_force_publish_admin_edition_path(edition, lock_version: edition.lock_version),
-        title: button_title,
-        class: "btn btn-default force-publish",
-      )
-    else
-      button_to(
-        "Publish",
-        confirm_publish_admin_edition_path(edition, lock_version: edition.lock_version),
-        title: button_title,
-        class: "btn btn-success publish",
-      )
-    end
-  end
-
-  def schedule_edition_form(edition, options = {})
-    button_title = "Schedule #{edition.title} for publication on #{l edition.scheduled_publication, format: :long}"
-
-    if options[:force]
-      button_to(
-        "Force schedule",
-        force_schedule_admin_edition_path(edition, lock_version: edition.lock_version),
-        data: { confirm: "Are you sure you want to force schedule this document for publication?" },
-        title: button_title,
-        class: "btn btn-warning",
-      )
-    else
-      button_to(
-        "Schedule",
-        schedule_admin_edition_path(edition, lock_version: edition.lock_version),
-        title: button_title,
-        class: "btn btn-success",
-      )
-    end
-  end
-
-  def unschedule_edition_button(edition)
```

Comment:
Have done a quick sense-check of a button at random (this one ^)

Looks like it was added here: https://github.com/alphagov/whitehall/commit/ed93c8e2a508936313fe9745f4832ea51ab065a7
Agree it no longer appears to be used. Nice find!